pkg/server/server.go

```diff
@@ -92,6 +93,13 @@ var (
 		NativeHistogramMaxBucketNumber:  100,
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"grpc_service", "grpc_method", "datastore_query_count", "dispatch_count", "consistency"})
+
+	checkResultCounterName = "check_result_count"
+	checkResultCounter     = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      checkResultCounterName,
+		Help:      "The total number of check requests by response result",
+	}, []string{allowedLabel})
```

Comment:
@adriantam Should we add `grpc_service` here to be consistent with other counters?

Reply:
No, this is meant only for check.  There is no allowed in any other  API.

Reply:
It's not clear to me when why we would even add that label in the first place for any metrics at all. Ostensibly its because we want user to be able to group all openfga metrics together. I mean these metrics are namespaced too so I'm not even sure why it was made into such a big deal in https://github.com/openfga/openfga/pull/1863. Can you please explain to me what purpose the `grpc_service` label serves?

Reply:
@adriantam @willvedd I looked deeper into this. `grpc_service` can take two values: the value for openfga, and `grpc.health.v1.health`. The latter only emits grpc metrics:
- grpc_server_handled.count
- grpc_server_handling_seconds.bucket
- grpc_server_handling_seconds.count
- grpc_server_handling_seconds.sum
- grpc_server_msg_received.count
- grpc_server_msg_sent.count
- grpc_server_started.count

As long as we have a prefix/namespace for each individual custom metric that we emit in the code, then I don't see a need to tag every metric with `grpc_service`.

Reply:
I _think_ the benefit of having the`grpc_service` metric is being able to aggregate all the metrics for OpenFGA within the context of a multi-service prometheus instance. Having the namespace helps distinguish between other services but it doesn't allow the ability to aggregate. However, that's somewhat of a guess and I'm not sure if operators actually find use in aggregating metrics like that.

Reply:
`grpc_service ` is a tag on a metric. For aggregation of metrics, one can use the namespace, which is `openfga`. So all metrics will be named `openfga.<something>`, I think that should be enough to aggregate 🤷‍♀️

Reply:
If two services were emitting metric `grpc_server_started.count` and you wanted to see their values in aggregate, you would want a tag like `grpc_service` on the metric.

Prefixing the metric name with the service name makes aggregation in a tool like DataDog rather difficult.